rmc/pipeline/two_breaks/run_batches.py

```diff
@@ -628,17 +628,27 @@ def main(args):
         default_python_image=args.docker_image,
         requester_pays_project=args.google_project,
     )
+    # Check if user specified list of numbers for batches
+    # These numbers are used to write output files for batch jobs
+    if args.counter:
+        count_list = args.counter.split(",")
 
     if args.under_threshold:
         section_groups = [
             sections_to_run[x : x + args.group_size]
             for x in range(0, len(sections_to_run), args.group_size)
         ]
         count = 1
+
         for group in tqdm(section_groups, unit="section group"):
-            logger.info("Working on group number %s...", count)
+            if count_list:
+                group_num = count_list[count - 1]
```

Comment:
Separately, I think there could be a better name for `check_for_successful_sections`. `get_unsuccessful_sections` maybe? Though that suggests that they failed

Reply:
> If args.counter length doesn't match the length of section_groups, this seems like it may not work

that's true! these numbers should match, so I'll add a check

> better name for check_for_successful_sections

hmm, would `get_sections_to_run` be better? checking for successful sections is how we determine which still need to be run, but good point that it doesn't seem like the best description within this script

Reply:
Sounds good!